packages/react-components/react-aria/src/activedescendant/useActiveDescendant.ts

```diff
@@ -143,6 +145,18 @@ export function useActiveDescendant<TActiveParentElement extends HTMLElement, TL
         }
       },
 
+      focusLastActive: () => {
+        if (!listboxRef.current || !lastActiveIdRef.current) {
+          return;
+        }
+
+        const target = listboxRef.current.querySelector<HTMLElement>(`#${lastActiveIdRef.current}`);
```

Comment:
```suggestion
        const target = listboxRef.current.ownerDocument.getElementById(lastActiveIdRef.current);
```

I think this will be faster for the browser than a query selector

Reply:
Specifically the focus function above this one

Reply:
Actually, while manually testing and writing the new unit test, `listboxRef.current.ownerDocument.getElementById` doesn't appear to work, whereas `listboxRef.current.querySelector<HTMLElement>` does